proto-google-cloud-spanner-v1/src/main/java/com/google/spanner/v1/TypeAnnotationCode.java

```diff
@@ -57,6 +57,8 @@ public enum TypeAnnotationCode implements com.google.protobuf.ProtocolMessageEnu
    * <code>PG_NUMERIC = 2;</code>
    */
   PG_NUMERIC(2),
+  /** <code>PG_JSONB = 3;</code> */
```

Comment:
nit: Looking at the comment above for `PG_NUMERIC`, I'm wondering if we should add more details in comment for `<code>PG_JSONB = 3;</code>`